src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Action
+const CHECKSTATUS = 'categories/CHECKSTATUS';
+const categories = [];
+
+// Reducer
+export default function reducerCategories(state = categories, action = {}) {
+  switch (action.type) {
+    case CHECKSTATUS:
+      return 'Under Construction';
+
+    default:
+      return state;
+  }
+}
+
+// Action Creation
+export function checkStatus() {
+  return { type: CHECKSTATUS };
+}
```

Comment:
- [ ] You have created the checkStatus action 👏 However, when the check status button is clicked, it is expected that you return a string showing 'under construction'. This doesn't happen yet. Please implement this requirement 🙏

Reply:
> * [ ]  You have created the checkStatus action clap However when the check status button is clicked, it is expected that you return a string showing 'under construction'. This doesn't happen yet. Please implement this requirement pray

You haven't implemented the above requirement suggested by the previous reviewer. Please fix the issue.